tensorflow/compiler/xla/service/gpu/cublas_cudnn.cc

```diff
@@ -51,6 +51,19 @@ const absl::string_view kCudnnConvBiasActivationForwardCallTarget =
 const absl::string_view kCudnnConvForwardCallTarget = "__cudnn$convForward";
 const absl::string_view kTriangularSolveCallTarget = "__cublas$triangularSolve";
 
+// fMHA call targets.
+const absl::string_view kCudnnfMHADefaultCallTarget = "__cudnn$fhmaDefault";
```

Comment:
Would it make more sense to use a unified call target and push those inside backend_config instead?

Reply:
Since `call_target` is an existing attribute, I felt it made sense to have separate call targets rather and adding another field to backend_config. Moreover, this is the convention followed by convolution as well.  Do you strongly suggest changing this?